### DISCUSSION TITLE
June Newletter

### DISCUSSION THREAD

**Discussion by briancl2** [briancl2/CustomerNewsletter#3] [Customer Newsletter]
June Newsletter

Brian's June Newsletter: Copilot Champions user list, JFrog Partnership, MSBuild Updates & Copilot Extensions Beta Launch

This is a personally curated newsletter for just my customers. The purpose is to pull together and share current resources, announcements, and events. I welcome your feedback, and please let me know if you would like to add or remove anyone from this list.

Highlights for June are Copilot Champions, JFrog Partnership, MSBuild Highlights & Copilot Extensions Beta Launch.

### Measuring the impact of GitHub Copilot - Copilot Champions
Last month I shared a number of resources to access GitHub’s Metrics API public beta. For July, I invite my Copilot customers to engage with the data in a deeper way through a few options:
* (EMU accounts only) Request your Copilot Champions list of power users. For accounts on Enterprise Managed Users (EMU), GitHub can provide two additional data points not accessible via API. These data points identify your top Copilot users to assist with training the rest of the user base on what is working well, tips and tricks, prompt engineering, etc. Some companies may choose to ask for volunteers amongst your Copilot Champions to host Lunch and Learns or staff regular office hours depending on the needs of your newer Copilot users that are still learning how to get the most out of the product. To request the your organization’s list of Copilot Champions, reply to this email.
* Using Power BI with GitHub Copilot Metrics API - [public repo](https://github.com/jasonmoodie/pbi-4-ghcopilot)
* If you aren’t already running regular surveys of your developer population to collect real-world data, automate Copilot developer surveys using this [public repo](https://github.com/github/copilot-survey-engine)

### GitHub and JFrog partner to unify code and binaries for DevSecOps
Together with JFrog, we’ve built an integration that includes intuitive navigation and traceability between source code and binaries, CI/CD with GitHub Actions and JFrog Artifactory, and a unified view of security findings across the software supply chain. 

<img width="1082" alt="Screenshot 2024-06-18 at 5 50 53 PM" src="https://github.com/user-attachments/assets/bd9a510e-8dd5-4dbf-9b63-8cc1bb928695">

With this new integration, we’ve integrated SSO, project role mapping and access management, and CI integration across both platforms. With centralized user IAM, users won’t need to worry about multiple logins, and changes made on one platform will automatically propagate to the other.

To find out more, join an upcoming joint GitHub & JFrog webinar: [Register here](https://jfrog.com/github-webinar/)

### Microsoft Build Updates for GitHub
Microsoft Build conference was held a few weeks ago, and there were a number of GitHub related announcements and sessions. 
* NEW Copilot Extensions Beta announced during Satya Nadella’s [Build Keynote](https://www.youtube.com/watch?v=8OviTSFqucI). Imagine you’re a developer who just got paged for a database-related incident. You’re trying to get context of the issue from a variety of tools such as a GitHub issue or audit logs in [DataStax](https://www.youtube.com/watch?v=3SmjSKsBRNk). With enough context, you start troubleshooting using tools like [Sentry](https://www.youtube.com/watch?v=irRaA4OZpGM) for error monitoring to learn more. Then you can figure out a solution, apply the fix, and deploy with [Azure](https://www.youtube.com/watch?v=Vf9zqtxbI9c) or [Octopus Deploy](https://www.youtube.com/watch?v=VO6fRiZwhCI). With the recently announced Copilot Extensions, you can do all of this within GitHub Copilot Chat without context switching. See more at the launch [blog post](https://github.blog/2024-05-21-introducing-github-copilot-extensions/). Additional resources:
    * A live demo of MongoDB, MSSql, Stripe Copilot Extensions - [Enhancing VS Code extensions with GitHub Copilot](https://build.microsoft.com/en-US/sessions/57efc1aa-83c0-45c5-b8c3-ad095478bb0a?source=sessions)
    * Build your own GitHub Copilot customer extension - [Extending GitHub Copilot](https://www.youtube.com/watch?v=RXaLlCeaBIA)
    * Upcoming live webinar with Atlassian, Elastic, and Split - [Register here ](https://resources.github.com/webinar-introducing-github-copilot-extensions-jun26/)
* Better Together - [Updated Azure Boards integration with GitHub](https://build.microsoft.com/en-US/sessions/cece7f04-c4b4-4fdc-b281-1d1508a3dcbf?source=sessions) - [blog post](https://devblogs.microsoft.com/devops/ab-links-on-github-pull-request-and-scale-improvements/)
* Measuring Impact - [How Accenture increases Developer Happiness and Build Success Rate with GitHub Copilot](https://www.youtube.com/watch?v=Xe0GBjZn5JE)
* How [GitHub builds GitHub on GitHub](https://build.microsoft.com/en-US/sessions/8dca5c87-37c3-4e9d-a6c9-54666b95c7ed?source=sessions)
* See all 28 on-demand GitHub related Build sessions [here](https://build.microsoft.com/en-US/sessions?search=github&sortBy=relevance)

### Other GitHub resources
* New Copilot Enterprise features in [VS Code](https://github.blog/changelog/2024-06-14-new-copilot-enterprise-features-in-vs-code-preview/) and [Visual Studio](https://github.blog/changelog/2024-06-13-new-copilot-enterprise-features-in-visual-studio-preview/)
* Industry Opinion from our friends at Stack Overflow - [Generative AI Is Not Going To Build Your Engineering Team For You](https://stackoverflow.blog/2024/06/10/generative-ai-is-not-going-to-build-your-engineering-team-for-you/)
* GitHub Copilot SOC2, Type 1 report - June 3, 2024 [blog post](https://github.blog/changelog/2024-06-03-github-copilot-compliance-soc-2-type-1-report-and-iso-iec-270012013-certification-scope/)
* Copilot Tips & Tricks [eBook](https://www.linkedin.com/posts/webmax_leveraging-github-copilot-chat-syntax-activity-7201545918851067904-F9EK/?utm_source=share&utm_medium=member_desktop)
* Brian’s personally curated Copilot Training & Tips [Youtube playlist](https://www.youtube.com/playlist?list=PLCiDM8_DsPQ1WJ5Ss3e0Lsw8EaijUL_6D), updated monthly

### Upcoming Webinars
* June 20: [Realizing the potential of GitHub Copilot Customer Roundtable](https://ms-workshops.cloudevents.ai/ms-innovation-workshops/events/5256530c-d3ac-4a10-8e5e-9677b4a934de)
* June 26: [Building Automation with GitHub Issues ](https://developer.microsoft.com/en-us/reactor/events/22755/)
* June 26: [Intro to GitHub Copilot](https://resources.github.com/intro-to-githubcopilot-june26th/)
* June 26: [Copilot Extensions with Atlassian, Elastic, and Split](https://resources.github.com/webinar-introducing-github-copilot-extensions-jun26/)
* July 18: [Copilot for Visual Studio Developers](https://developer.microsoft.com/en-us/reactor/events/22918/)
* July 24: [GitHub Copilot Getting Started for new users](https://developer.microsoft.com/en-us/reactor/events/22980/)
* August 7: [GitHub Public Roadmap Q3](https://resources.github.com/webcasts/github-roadmap-webinar-q3/)

### Upcoming In-person events
* Irvine, CA - June 27 - GitHub + Microsoft Connect half-day DevSecOps AI event - [Registration](https://resources.github.com/github-and-msft-connect-june2024/)
* San Francisco - Oct 29-30 - [Early Bird Tickets](https://githubuniverse.com/#tickets) available through July 8

**Comment by alwell-kevin** [comment]
Terrific newsletter. Thank you Brian! Please keep them coming.

**Comment by llamountain** [comment]
this is incredible brian!  you're awesome for sharing.  thank you!  🫶

**Comment by LauraRod32** [comment]
@briancl2 This is amazing! Thank you for sharing this with us.
